### PR TITLE
[MON-1941]: Runbooks for FailedToSendAlerts alert

### PR DESCRIPTION
**Pull request by sthaha**
Adds two Runbooks for `AlertmanagerFailedReload` and `AlertmanagerClusterFailedToSendAlerts` alerts

### CODE REVIEW

**Comment by sthaha** [comment]
cc: @simonpasquier @wking could you please take a look?

**Review by bison — APPROVED** [review]
/lgtm

This looks like a good start to me. Hopefully over time people can add specific real-world examples.

File: alerts/cluster-monitoring-operator/AlertmanagerClusterFailedToSendAlerts.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# AlertmanagerClusterFailedToSendAlerts
+
+## Meaning
+
+The alert `AlertmanagerClusterFailedToSendAlerts` is triggered when **all**
+Alertmanager instances in a cluster has consistently failed to send
+notifications to a critical integration.
```
Comment (simonpasquier) [inline alerts/cluster-monitoring-operator/AlertmanagerClusterFailedToSendAlerts.md:7]: we don't separate between critical/non-critical integrations in OCP.

```suggestion
notifications to an integration.
```

File: alerts/cluster-monitoring-operator/AlertmanagerClusterFailedToSendAlerts.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Impact
+
+Alerts to the integration are not delivered.
```
Comment (simonpasquier) [inline alerts/cluster-monitoring-operator/AlertmanagerClusterFailedToSendAlerts.md:11]:
```suggestion
A fraction of alert notifications to the integration are not delivered.
```

File: alerts/cluster-monitoring-operator/AlertmanagerClusterFailedToSendAlerts.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Check the logs of the `alertmanager-main` pods in the `openshift-monitoring`
+namespace:
+
+```console
+$ oc -n openshift-monitoring logs -l 'alertmanager=main'
+```
+
+## Mitigation
+
+The resolution depends on the particular issue reported in the logs.
```
Comment (simonpasquier) [inline alerts/cluster-monitoring-operator/AlertmanagerClusterFailedToSendAlerts.md:24]: Maybe list a few reasons why the alert might exist like unreachable endpoint, misconfigured address or credentials, ...?

File: alerts/cluster-monitoring-operator/AlertmanagerFailedToSendAlerts.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Meaning
+
+The alert `AlertmanagerFailedToSendAlerts` is triggered when any of the
+Alertmanager instances of the cluster monitoring stack has consistently
+failed to send notifications to an integration.
+
+## Impact
+
+Some of the alerts are not delivered.
```
Comment (simonpasquier) [inline alerts/cluster-monitoring-operator/AlertmanagerFailedToSendAlerts.md:11]: I'd rather say "notifications" rather than "alerts".

**Pull request merged**